src/components/Table/index.tsx

```diff
@@ -275,12 +276,13 @@ export default function Table<D extends object>({
 
     const columns = useMemo(() => {
         const columnsFiltered = columnDefinitions.filter((column: Column<D>) => showColumns[column.id || '']);
-        if (!disableRowSelect) {
+        if (disableRowSelect !== true) {
+            const disableRowSelectRows = new Set(disableRowSelect === false ? [] : disableRowSelect);
```

Comment:
cannot it be
```js
const disableRowSelectRows = new Set(disableRowSelect ?? []);
```

Reply:
`false ?? []` will evaluate to `false` which isn't what we want :) I could swap it to `disableRowSelect || []` I suppose?

Reply:
@cogwirrel yeah!) That what i wanted to write))